### ISSUE TITLE
'ImportError: cannot import name extract' on Ubuntu 14.10

### ISSUE THREAD

**Issue by ekajogja** [mbusb/multibootusb#16]
Installation succeed (python-psutils and python-qt4 already installed) got this error while running on Ubuntu 14.10:
```
Traceback (most recent call last):
  File "/usr/local/bin/multibootusb", line 26, in <module>
    from scripts import extract
ImportError: cannot import name extract

**Comment by mbusb** [comment]
Comment or remove the line "from scripts import extract". The error is thrown because the file does not exist.

**Comment by ekajogja** [comment]
Commented the line, re-run, got this error:
```
Cleaning old multibootusb directory.
Extracting syslinux to multibootusb directory.
Copying persistence data to multibootusb directory.
Traceback (most recent call last):
  File "/usr/local/bin/multibootusb", line 96, in <module>
    shutil.copytree(resource_path(os.path.join("tools", "persistence_data")), os.path.join(mbusb_dir, "persistence_data"))
  File "/usr/lib/python2.7/shutil.py", line 171, in copytree
    names = os.listdir(src)
TypeError: coercing to Unicode: need string or buffer, NoneType found

**Comment by mbusb** [comment]
Ok. Got it. Pushing my development version. This should take care of all issues.

**Comment by mbusb** [comment]
Pushed new files. Can you check the new source code?

**Comment by ekajogja** [comment]
Pulled new code > reinstall > run > sudo prompt > filled sudo pass, got this:
```
/usr/bin/sudo
Using PyUdev for detecting USB drives...
Using PyUdev for detecting USB details...
Name of the USB partition is multiboot
Traceback (most recent call last):
  File "/usr/local/bin/multibootusb", line 345, in <module>
    window = AppGui()
  File "/usr/local/bin/multibootusb", line 89, in __init__
    self.add_device()
  File "/usr/local/bin/multibootusb", line 109, in add_device
    self.onComboChange()
  File "/usr/local/bin/multibootusb", line 114, in onComboChange
    usb_details = usb.usb_details(str(self.ui.comboBox.currentText()))
  File "/usr/local/lib/python2.7/dist-packages/scripts/usb.py", line 230, in usb_details
    'total_size': bytes2human(disk_usage(mount_point_strip).total),
  File "/usr/local/lib/python2.7/dist-packages/scripts/usb.py", line 14, in disk_usage
    st = os.statvfs(mount_path)
OSError: [Errno 2] No such file or directory: ''

**Comment by mbusb** [comment]
Is your drive mounted?

**Comment by mbusb** [comment]
Checked at my end. You need to mount drive before using multibootusb.

**Comment by ekajogja** [comment]
Sorry, my bad! This is what I got after usb mounted:
```
/usr/bin/sudo
[sudo] password for eka: Using PyUdev for detecting USB drives...
Using PyUdev for detecting USB details...
Name of the USB partition is multiboot
syslinux.cfg does not exist for list update.
Cleaning old multibootusb directory.
Extracting syslinux to multibootusb directory...
Traceback (most recent call last):
  File "/usr/local/bin/multibootusb", line 345, in <module>
    window = AppGui()
  File "/usr/local/bin/multibootusb", line 91, in __init__
    gen_fun.prepare_mbusb_dir()
  File "/usr/local/lib/python2.7/dist-packages/scripts/gen_fun.py", line 128, in prepare_mbusb_dir
    with zipfile.ZipFile(resource_path(os.path.join("tools", "syslinux", "syslinux_linux.zip")), "r") as z:
  File "/usr/lib/python2.7/zipfile.py", line 770, in __init__
    self._RealGetContents()
  File "/usr/lib/python2.7/zipfile.py", line 807, in _RealGetContents
    endrec = _EndRecData(fp)
  File "/usr/lib/python2.7/zipfile.py", line 208, in _EndRecData
    fpin.seek(0, 2)
AttributeError: 'NoneType' object has no attribute 'seek'

**Comment by mbusb** [comment]
Corrected the usb mount issue and pushed the changes. But I am unable to reproduce the other issue `File "/usr/local/lib/python2.7/dist-packages/scripts/gen_fun.py", line 128, in prepare_mbusb_dir
    with zipfile.ZipFile(resource_path(os.path.join("tools", "syslinux", "syslinux_linux.zip")), "r") as z:`

I have downloaded the latest source code and no issue here. Can you reproduce and tell me exactly what you did?

**Comment by mbusb** [comment]
I can confirm you that there is no issue with latest source. This is what I followed. 
1. Booted in to ubuntu 14.10 using live usb (created with multibootusb).
2. Downloaded latest source code and extracted
3. Then moved in tt the extracted directory.
4. Typed ./multibootusb and enter.
5. Follwing is the result:-
`ubuntu@ubuntu:~/Downloads/multibootusb-master$ ./multibootusb
/usr/bin/sudo
Using PyUdev for detecting USB drives...
Falling back to Udisks2..
Using PyUdev for detecting USB details...
Falling back to UDisks2 for detecting USB details...
Falling back to UDisks as a last choice for getting USB details...
Error detecting USB details.
Device mounted.
Device ststus is /cdrom
Using PyUdev for detecting USB details...
Falling back to UDisks2 for detecting USB details...
Falling back to UDisks as a last choice for getting USB details...
Error detecting USB details.
Device mounted.
Device ststus is /cdrom
Cleaning old multibootusb directory.
Extracting syslinux to multibootusb directory...
Extracting syslinux modules to multibootusb directory...
/home/ubuntu/Downloads/multibootusb-master/tools/syslinux/syslinux_modules.zip
Copying persistence data to multibootusb directory.
Closing multibootusb...
ubuntu@ubuntu:~/Downloads/multibootusb-master$`

I think you are not moving in to the downloaded source directory. When running from source code, you must be top of source code directory.
